src/Typeahead.vue

```diff
@@ -13,7 +13,7 @@
       @keydown.esc="reset"
       @blur="showDropdown = false"
     />
-    <ul class="dropdown-menu" v-el:dropdown>
+    <ul class="dropdown-menu search-dropdown-menu" v-el:dropdown>
```

Comment:
Instead of modifying Typeahead.vue, let's add this in Searchbar.vue:

```js
mounted() {
  this.$els.dropdown.classList.add('search-dropdown-menu');
},
```

You can add it between `extends` and `partials`.

Reply:
[`mounted`](https://vuejs.org/v2/api/#mounted) doesn't seem to work in Vue 1 (even with `this.$nextTick`); let's use [`ready`](https://v1.vuejs.org/api/#ready).

Reply:
Sure